tests/commands/harddrive.py

```diff
@@ -75,5 +70,17 @@ def runTest(self):
         # unknown option
         self.assert_parse_error("harddrive --unknown=value")
 
+
+class F33_TestCase(FC3_TestCase):
```

Comment:
If you don't base it on the the FC3 testcase then you won't have to change the FC3 code. You'll have to duplicate the rest of the tests in here though.

Previous versions of kickstart need to continue to support it (eg. when passing --version=F27), so we shouldn't change the older tests.

Reply:
I leaved the old tests untouched and make these independent. Hope I did it correctly. :)

Reply:
I've also tried to extend the "compare" test. If there is other way how to do this correctly I'll change it appropriately.